python/doc/examples/meta_modeling/general_purpose_metamodels/plot_distribution_linear_regression.py

```diff
@@ -0,0 +1,245 @@
+    elif estimator == "standard-deviation":
+        sample_estimator.setDescription([r"$\hat{\sigma}$"])
+    else:
+        raise ValueError("Unknown estimator %s" % (estimator))
+    mean_sample_estimator = sample_estimator.computeMean()
+
+    graph = ot.KernelSmoothing().build(sample_estimator).drawPDF()
+    graph.setLegends(["Sample"])
+    graph.setTitle(
+        "Size = %d, True S.D. = %.4f, Mean = %.4f, Rep. = %d"
+        % (
+            sample_size,
+            true_standard_deviation,
+            mean_sample_estimator[0],
+            repetitions_size,
+        )
+    )
+    return graph
+
+
+# %%
+# Distribution of the variance estimator
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# We first consider the estimation of the variance :math:`\sigma^2`.
+# In the next cell, we consider a sample size equal to :math:`n = 6` with
+# :math:`p = 3` parameters.
+# We use :math:`r = 1000` repetitions.
+
+
+repetitions_size = 1000
+true_standard_deviation = 0.1
+sample_size = 6
+coefficients = ot.Point([3.0, 2.0, -1.0])
+estimator = "variance"
+view = otv.View(
+    plot_sample_by_kernel_smoothing(
+        sample_size, true_standard_deviation, coefficients, estimator, repetitions_size
+    ),
+    figure_kw={"figsize": (6.0, 3.5)},
+)
+pl.subplots_adjust(bottom=0.25)
+
+
+# %%
+# If we use a sample size equal to :math:`n = 6` with
+# :math:`p = 3` parameters, the distribution is not symmetric.
+# The mean of the observations of the sample variances still is close to
+# the true value, that is :math:`0.1^2 = 0.01`.
+
+# %%
+# Then we increase the sample size :math:`n`.
+
+repetitions_size = 1000
+true_standard_deviation = 0.1
+sample_size = 100
+coefficients = ot.Point([3.0, 2.0, -1.0])
+view = otv.View(
+    plot_sample_by_kernel_smoothing(
+        sample_size, true_standard_deviation, coefficients, estimator, repetitions_size
+    ),
+    figure_kw={"figsize": (6.0, 3.5)},
+)
+pl.subplots_adjust(bottom=0.25)
+
+
+# %%
+# If we use a sample size equal to :math:`n = 6` with
+# :math:`p = 3` parameters, the distribution is almost symmetric and
+# almost Gaussian.
+
+# %%
+# Distribution of the standard deviation estimator
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# We now consider the estimation of the standard deviation :math:`\sigma`.
+
+
+repetitions_size = 1000
+true_standard_deviation = 0.1
+sample_size = 6
+coefficients = ot.Point([3.0, 2.0, -1.0])
+estimator = "standard-deviation"
+view = otv.View(
+    plot_sample_by_kernel_smoothing(
+        sample_size, true_standard_deviation, coefficients, estimator, repetitions_size
+    ),
+    figure_kw={"figsize": (6.0, 3.5)},
+)
+pl.subplots_adjust(bottom=0.25)
+
+
+# %%
+# If we use a sample size equal to :math:`n = 6` with
+# :math:`p = 3` parameters, we see that the distribution is almost symmetric.
+# We notice a slight bias, since the mean of the observations of the
+# standard deviation is not as close to the true value 0.1
```

Comment:
Couldn't you plot the true value as a vertical line? This would make the bias more apparent.

Reply:
Like this?

![image](https://github.com/openturns/openturns/assets/31351465/19f64c29-0842-441e-a97a-90831376accd)

Indeed: I will do this.

Reply:
Indeed, but maybe with a smaller width ^^

Reply:
Fixed at c30767685.

![image](https://github.com/openturns/openturns/assets/31351465/85d5e8f0-57a7-4ec4-b648-49f6be87a95c)